add eep-f6-10-00

huwylphi opened the pull request:
Mechanical window handle (HOPPE)

Holger-Will commented:
Thanks for adding this! 

i am currently writing a generic eep resolver base on this data:
https://github.com/node-enocean/eep-spec

so there might be some major changes of the retured json structure ahead (planned for 3.0)! this will ultimately render your patch obsolete. But i do not know how long it will take, so happily accepting your patch.

Holger-Will commented:
merged, moved your file to the right place, added some test for this eep, and published to npm. new version is 2.1.0

huwylphi commented:
Hi, thank you for this infos. Sorry for the wrong place of the file. I'm
still beginner with github... Can't wait to see your next releases :)


On Sat, May 6, 2017 at 7:37 AM, Holger Will <notifications@github.com>
wrote:

> merged, moved your file to the right place, added some test for this eep,
> and published to npm. new version is 2.1.0
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/node-enocean/node-enocean/pull/26#issuecomment-299617534>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADAlGmNzgwOGoy7iNbw2x3pWSAa11RO7ks5r3AcmgaJpZM4NSasw>
> .
>
